Tests/MoyaTests/NetworkLoggerPluginSpec.swift

```diff
@@ -11,19 +11,12 @@ final class NetworkLoggerPluginSpec: QuickSpec {
 
         let customLoggerOutput: NetworkLoggerPlugin.Configuration.OutputType = { log += $1.joined() }
 
-        let plugin = NetworkLoggerPlugin(configuration: .init(output: customLoggerOutput,
-                                                              logOptions: .verbose))
-
-        let pluginWithCurl = NetworkLoggerPlugin(configuration: .init(output: customLoggerOutput,
-                                                                      logOptions: [.formatRequestAscURL]))
-
-        let pluginWithRequestDataFormatter = NetworkLoggerPlugin(configuration: .init(output: customLoggerOutput,
-                                                                                      requestDataFormatter: { _ in return "formatted request body" },
-                                                                                      logOptions: .verbose))
-
-        let pluginWithResponseDataFormatter = NetworkLoggerPlugin(configuration: .init(output: customLoggerOutput,
-                                                                                       responseDataFormatter: { _ in return "formatted response body" },
-                                                                                       logOptions: .verbose))
+        let plugin: NetworkLoggerPlugin = {
```

Comment:
can you rename the `plgn` to something else? `plugin` might work with shadowing variables but if not, maybe `networkPlugin`?